airflow/contrib/operators/sqoop_operator.py

```diff
@@ -59,12 +66,18 @@ def __init__(self,
                  properties=None,
                  hcatalog_database=None,
                  hcatalog_table=None,
+                 create_hcatalog_table=False,
+                 extra_import_options=None,
+                 extra_export_options=None,
```

Comment:
adding options to pass in dictionaries of k-v pairs of the sqoop option and the value want to pass with it for both the import and export commands

Reply:
👍